extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -191,6 +192,12 @@ describe('amp-a4a', () => {
     expect(child.src).to.match(/^https?:[^?#]+nameframe(\.max)?\.html/);
     const nameData = child.getAttribute('name');
     expect(JSON.parse.bind(null, nameData), nameData).not.to.throw(Error);
+    const attributes = JSON.parse(nameData);
+    expect(attributes).to.be.ok;
+    expect(attributes._context).to.be.ok;
+    if (!attributes._context.amp3pSentinel) {
+      expect(attributes._context.sentinel).to.be.ok;
+    }
```

Comment:
The two cases for sentinel naming will collapse down very soon, so I didn't want to modify the other tests too much. I can add additional logic in here as suggested for making sure it doesn't have both keys and that it matches the regex